go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/grafana/grafana-api-golang-client
+module github.com/justinTM/grafana-api-golang-client
```

Comment:
Can you revert that change?

Reply:
97fa6d8c2e586d7d12011bf87fbb6679cce7ccec